lib/comfortable_mexican_sofa/extensions/is_taggable.rb

```diff
@@ -8,7 +8,7 @@ module ClassMethods
         # Comfy already define #tags and #tags= methods in some models, so let's call :keywords to the Tag associated
         # instances of a model to avoid naming collisions!!!
         # Creates the appropriate AR associations, scopes and instance methods.
-        def is_taggable
+        def is_taggable # rubocop:disable Style/PredicateName
```

Comment:
@neoeno Actually this is a good PR. Please, feel free to have this ideas. I like this initiative to solve the problems on the cms codebase and definitely I will help you on this as much as I can.
The thing I was just trying to say before, that tricking rubocop was not so good. 
I will have another meething on the afternoon, but let combine to pair on this issue. I will be glad to help.